src/components/page/AppPage.js

```diff
@@ -60,17 +85,18 @@ class AppPage extends React.Component {
                 { this.state.clicked ? <Spinner/>: ""}
 
                 <TextWrapper>
-                    <GradationText style= {{marginTop:"56px"}} fontSize={2.6} fontWeight={'bold'}>
-                        PythonQuizFlex
-                    </GradationText>
+                    <LogoImg/>
+
+                    <GradationTitle>
```

Comment:
Can it be styled component?

Reply:
I deleted the GradationTitle and changed the direction to reuse the GradationText! Done